CONTRIBUTING.md

```diff
@@ -6,16 +6,13 @@
 2. Fork the repo
 3. Clone the fork
 4. Install the dependencies: `npm i`
-5. Run `npm run build:lib`
-6. Run `npm link`
-7. Run `npm link @maplibre/maplibre-gl-directions`
-8. Run `npm run build:lib`
```

Comment:
Is the second build:lib not needed in the command in the package.sjon file?

Reply:
@wipfli I think that no, one time should be enough